python/ray/_raylet.pyx

```diff
@@ -1252,7 +1256,6 @@ cdef report_streaming_generator_output(
     cdef:
```

Comment:
We should update the docstring here.

Reply:
oh no. if you read the docstring, it has Returns:... which is not applicable anymore since this PR removes the return value.

Reply:
Good catch!